web/src/LogActions.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect } from "react"
+import { useStorageState } from "react-storage-hooks"
+import styled from "styled-components"
+import { incr } from "./analytics"
+import ClearLogs from "./ClearLogs"
+import { Color, FontSize, mixinResetButtonStyle } from "./style-helpers"
+
+export const LogFontSizeScaleLocalStorageKey = "tilt.global.log-font-scale"
+export const LogFontSizeScaleCSSProperty = "--log-font-scale"
+export const LogFontSizeScaleMinimumPercentage = 10
+
+const LogActionsGroup = styled.div`
+  margin-left: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`
+
+const FontSizeControls = styled.div`
+  color: ${Color.gray6};
+  vertical-align: middle;
+  display: flex;
+  flex-wrap: nowrap;
+`
+
+const FontSizeControlsDivider = styled.div`
```

Comment:
it feels weird to do a divider like this...usually i see plain html content, or using ":after" on a button element, but not ":after" on a divider element

Reply:
Made more standard in e86d044c5df84e78539232797a92bd5578dfd32a